apps/src/templates/instructions/DocumentationTab.jsx

```diff
@@ -1,26 +1,101 @@
-import React from 'react';
+import React, {useState} from 'react';
 import {connect} from 'react-redux';
 import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
 import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
 import {useFetch} from '@cdo/apps/util/useFetch';
+import ProgrammingClassOverview from '../codeDocs/ProgrammingClassOverview';
+import {TextLink} from '@dsco_/link';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+const DEFAULT_CLASS = 'Java Basics - The main Method';
```

Comment:
I don't love having to hard code this, open to suggestions on how we define a default.

Reply:
Is there an `id` or `key` on the model we could key off of?

Reply:
We could use the key for this (which I think would be better in case we translate these pages or something), but I still don't love hardcoding that. But I can't think of a better option either...